web3.md

```diff
@@ -285,11 +286,13 @@ WEB3 JSON RPC
          <batch> [ _ ] </batch>
 
     syntax KItem ::= #rpcResponseSuccess       ( JSON                )
+                   | #rpcResponseRevert        ( JSON , JSON         )
```

Comment:
Since this doesn't do anything specific to `Revert`, let's rename it to `#rpcResponseSuccessWithError`.